docs_nnx/guides/linen_to_nnx.rst

```diff
@@ -595,23 +601,23 @@ For more information on NNX transforms, check out the `Transforms Guide <https:/
   model = MLP(64, num_layers=5, rngs=nnx.Rngs(0))
 
 
-There are a few other details to explain in this example:
+There are a few other details to explain in the Flax NNX example above:
 
-* **What is that `nnx.split_rngs` decorator?** NNX transforms are completely agnostic of RNG state, which makes them behave more like JAX transforms but diverge from the Linen transforms that handle RNG state. To regain this functionality, the ``nnx.split_rngs`` decorator allows you to split the ``Rngs`` before passing them to the decorated function and 'lower' them afterwards so they can be used outside.
+* **The `@nnx.split_rngs` decorator** Flax NNX transforms are completely agnostic of PRNG state, which makes them behave more like JAX transforms but diverge from the Flax Linen transforms that handle PRNG state. To regain this functionality, the ``nnx.split_rngs`` decorator allows you to split the ``nnx.Rngs`` before passing them to the decorated function and 'lower' them afterwards, so they can be used outside.
```

Comment:
Add a `:` at the end of `**The `@nnx.split_rngs` decorator**`

Reply:
Good catch, done.